### ISSUE TITLE
Metrics don't seem to work when docker-compose

### ISSUE THREAD

**Issue by ivanklee86** [Unleash/unleash-docker#15]
I ran into this oddball issue while taking a swing @ https://github.com/Unleash/unleash-client-python/issues/16

**Steps to Reproduce**
1. Start up Unleash + Postgres using `docker-compose build` and `docker-compose up`.
2. Create a demo feature flag:
3. Create a silly simple script to check feature flag once every ten seconds (or something similar.
```python
import time
from UnleashClient import UnleashClient

# ---
import logging
import sys

root = logging.getLogger()
root.setLevel(logging.DEBUG)

handler = logging.StreamHandler(sys.stdout)
handler.setLevel(logging.DEBUG)
formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
handler.setFormatter(formatter)
root.addHandler(handler)
# ---


my_client = UnleashClient(
    url="http://localhost:4242/api",
    app_name="pyIvan"
)

my_client.initialize_client()

while True:
    time.sleep(10)
    print(my_client.is_enabled("Demo"))
```

Repeat steps 1-3 using an Unleash server instance backed by a Postgres database.

**Issue Observed**
When using docker compose, the feature flag metric seems stuck on 0/0.

<img width="983" alt="Screen Shot 2019-04-19 at 6 57 59 AM" src="https://user-images.githubusercontent.com/14824978/56421520-df140080-6270-11e9-98f7-eb378f6fd2c0.png">

exec-ing into the Postgres container seems to show that the metrics data is present in the database correctly.  (This was from a test run I did yesterday AM.)

```
postgres=# select * from client_metrics
postgres-# ;
 id  |          created_at           |                                                                                                metrics
-----+-------------------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
  46 | 2019-04-18 11:51:56.718434+00 | {"appName":"pyIvan","instanceId":"unleash-client-python","bucket":{"start":"2019-04-18T07:50:56.456Z","stop":"2019-04-18T07:51:56.672Z","toggles":{"Demo":{"yes":5,"no":0}}}}
  47 | 2019-04-18 11:52:56.720726+00 | {"appName":"pyIvan","instanceId":"unleash-client-python","bucket":{"start":"2019-04-18T07:51:56.706Z","stop":"2019-04-18T07:52:56.672Z","toggles":{"Demo":{"yes":6,"no":0}}}}
  48 | 2019-04-18 11:53:56.714373+00 | {"appName":"pyIvan","instanceId":"unleash-client-python","bucket":{"start":"2019-04-18T07:52:56.699Z","stop":"2019-04-18T07:53:56.667Z","toggles":{"Demo":{"yes":6,"no":0}}}}
  49 | 2019-04-18 11:54:56.687185+00 | {"appName":"pyIvan","instanceId":"unleash-client-python","bucket":{"start":"2019-04-18T07:53:56.695Z","stop":"2019-04-18T07:54:56.670Z","toggles":{"Demo":{"yes":6,"no":0}}}}
  79 | 2019-04-18 11:58:35.136196+00 | {"appName":"pyIvan","instanceId":"unleash-client-python","bucket":{"start":"2019-04-18T07:57:34.957Z","stop":"2019-04-18T07:58:35.107Z","toggles":{"Demo":{"yes":5,"no":0}}}}
  80 | 2019-04-18 11:59:35.133494+00 | {"appName":"pyIvan","instanceId":"unleash-client-python","bucket":{"start":"2019-04-18T07:58:35.151Z","stop":"2019-04-18T07:59:35.107Z","toggles":{"Demo":{"yes":6,"no":0}}}}
  81 | 2019-04-18 12:00:35.146794+00 | {"appName":"pyIvan","instanceId":"unleash-client-python","bucket":{"start":"2019-04-18T07:59:35.135Z","stop":"2019-04-18T08:00:35.104Z","toggles":{"Demo":{"yes":6,"no":0}}}}
  82 | 2019-04-18 12:01:35.137373+00 | {"appName":"pyIvan","instanceId":"unleash-client-python","bucket":{"start":"2019-04-18T08:00:35.157Z","stop":"2019-04-18T08:01:35.104Z","toggles":{"Demo":{"yes":6,"no":0}}}}
  83 | 2019-04-18 12:02:35.172807+00 | {"appName":"pyIvan","instanceId":"unleash-client-python","bucket":{"start":"2019-04-18T08:01:35.144Z","stop":"2019-04-18T08:02:35.110Z","toggles":{"Demo":{"yes":6,"no":0}}}}
  84 | 2019-04-18 12:03:35.13187+00  | {"appName":"pyIvan","instanceId":"unleash-client-python","bucket":{"start":"2019-04-18T08:02:35.184Z","stop":"2019-04-18T08:03:35.106Z","toggles":{"test":{"yes":0,"no":0},"Demo":{"yes":6,"no":0}}}}
  85 | 2019-04-18 12:04:35.13085+00  | {"appName":"pyIvan","instanceId":"unleash-client-python","bucket":{"start":"2019-04-18T08:03:35.134Z","stop":"2019-04-18T08:04:35.105Z","toggles":{"test":{"yes":0,"no":0},"Demo":{"yes":6,"no":0}}}}
 115 | 2019-04-18 12:09:09.59915+00  | {"appName":"pyIvan","instanceId":"unleash-client-python","bucket":{"start":"2019-04-18T08:08:09.475Z","stop":"2019-04-18T08:09:09.554Z","toggles":{"Demo":{"yes":5,"no":0},"test":{"yes":0,"no":0}}}}
 116 | 2019-04-18 12:10:09.592847+00 | {"appName":"pyIvan","instanceId":"unleash-client-python","bucket":{"start":"2019-04-18T08:09:09.606Z","stop":"2019-04-18T08:10:09.559Z","toggles":{"Demo":{"yes":6,"no":0},"test":{"yes":0,"no":0}}}}
(13 rows)

postgres=# select * from client_applications;
          app_name          |          created_at           |          updated_at           | seen_at |                                                               strategies                                                               | description | icon | url | color
----------------------------+-------------------------------+-------------------------------+---------+----------------------------------------------------------------------------------------------------------------------------------------+-------------+------+-----+-------
 unleash-python-integration | 2019-04-01 11:13:03.873981+00 | 2019-04-01 11:13:03.873981+00 |         | ["applicationHostname","default","gradualRolloutRandom","gradualRolloutSessionId","gradualRolloutUserId","remoteAddress","userWithId"] |             |      |     |
 pyIvan                     | 2019-04-01 11:13:03.873981+00 | 2019-04-18 12:08:09.531497+00 |         | ["applicationHostname","default","gradualRolloutRandom","gradualRolloutSessionId","gradualRolloutUserId","remoteAddress","userWithId"] |             |      |     |
(2 rows)

postgres=# select * from client_instances;
 app_name |      instance_id      |     client_ip     |           last_seen           |          created_at           |         sdk_version
----------+-----------------------+-------------------+-------------------------------+-------------------------------+-----------------------------
 pyIvan   | unleash-client-python | ::ffff:172.19.0.1 | 2019-04-18 12:10:09.619072+00 | 2019-04-18 11:50:56.566464+00 | unleash-client-python:2.1.0
(1 row)

postgres=#
```

Oddly, if I run a local Unleash server pointed at a remote Postgres database (a free ElephantSQL database), it work totally fine!

**Comment by ivarconr** [comment]
Thanks for reporting. The docker-compose setup has never been intended to be a production ready setup. 

What version of unleash-server did you use in your setup? I introduced a metrics-startup fix in `3.2.3`, which would result in the scenario you describe if the database is unavailable at startup.

**Comment by ivanklee86** [comment]
Whoops, I think this was actually caused by [this bug](https://github.com/Unleash/unleash-client-python/pull/17) in the Python client.

On the bright side, this my main ??? for implementing e2e tests. =)
